generator.yml

```diff
@@ -263,10 +263,16 @@ builds:
   #############
   #  Android  #
   #############
+  - {<< : *arm32_allmod,      << : *android-mainline, << : *llvm_full,       boot: false, llvm_version: *llvm_tot}
+  - {<< : *arm32_v7_t,        << : *android-mainline, << : *llvm_full,       boot: false, llvm_version: *llvm_tot}
```

Comment:
We should be able to boot the Thumb2 ones, right?

Reply:
good catch! PTAL